### DISCUSSION TITLE
Installing hacmini.pkg 3.2.2 on Monterey installation U disk shows installation failed

### DISCUSSION THREAD

**Discussion by GitMasterArm** [osy86/HaC-Mini#1049] [Q&A] [answered]
**Describe the bug**
After I made a Monterey installation U disk, the installation of hacmini.pkg 3.2.2 showed that the installation failed, and the xcode-select command line was installed. The same error is reported when installing other versions of hacmini.pkg, please help.

**Expected behavior**
![image](https://user-images.githubusercontent.com/45396912/201117455-831880ab-5615-449a-85e1-95f5f02e00c8.png)
**Environment:**
 - Running OSX version: monterey 12.6.1
 - Target drive OSX version: monterey 12.6.1
 - HaC Mini installer version: v3.2.2

**Additional context**

**Logs**
Please attach `/var/log/install.log`
[install.log](https://github.com/osy/HaC-Mini/files/9981755/install.log)

**Comment by Medizinmann-MD** [comment, marked as answer]
Did you try to run 'sudo xcodebuild -license' as suggested in the logfile?

**Comment by GitMasterArm** [comment]
> 
OK, I'll try.

**Comment by S1999U2** [comment]
For me the issue was solved by unchecking "Secure Boot" in the installer options.
